test/data.jl

```diff
@@ -271,25 +277,25 @@ module TestData
     #               NullableArray(Nullable{String}["x", "x", "y", "y",
     #                                              "x", "x", "x", "x", "x", "y",
     #                                              Nullable(), "y"])
-
-    srand(1)
-    function spltdf(d)
-        d[:x1] = map(x -> get(x)[1], d[:a])
-        d[:x2] = map(x -> get(x)[2], d[:a])
-        d[:x3] = map(x -> get(x)[3], d[:a])
-        d
-    end
-    df1 = DataTable(
-        a = ["abc", "abx", "axz", "def", "dfr"],
-        v1 = randn(5)
-    )
-    df1 = spltdf(df1)
-    df2 = DataTable(
-        a = ["def", "abc","abx", "axz", "xyz"],
-        v2 = randn(5)
-    )
-    df2 = spltdf(df2)
-
+    # 
```

Comment:
The commits says `get` is removed, but you just uncommented it.

Anyway, I think these failures only happen with NullableArrays from master (I got trapped too), so please put them in a separate PR that we can merge as soon as a new release is tagged.